src/graphql/schema/uuid/user/resolvers.ts

```diff
@@ -21,7 +21,12 @@
  */
 import { pipeable, either } from 'fp-ts'
 
-import { AbstractUuidPayload, encodePath, UserPayload } from '..'
+import {
+  AbstractUuidPayload,
+  createUuidResolvers,
+  encodePath,
+  UserPayload,
+} from '..'
```

Comment:
```suggestion
import {
  AbstractUuidPayload,
  createUuidResolvers
} from '../abstract-uuid'
import {
  encodePath,
  UserPayload,
} from '..'
```

This solves the issue with the import and the tests (I have no idea why -> might be a circular dependency thing)